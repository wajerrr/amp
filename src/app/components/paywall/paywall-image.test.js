import React from 'react';
import renderer from 'react-test-renderer';
import PaywallImage from './paywall-image';

describe('PaywallImage', () => {
  it('should match the snapshot', () => {
    const paywallImage = renderer
      .create(
        <PaywallImage
          data={{
            id: 'regwall-footer',
            type: 'image',
            content: {
              src:
                'https://i.piano.io/managedservices/theeconomist/regwall-product.png',
              alt: 'paywall-image',
            },
          }}
        />
      )
      .toJSON();
    expect(paywallImage).toMatchSnapshot();
  });
});
