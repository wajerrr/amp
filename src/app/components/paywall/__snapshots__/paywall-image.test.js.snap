// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaywallImage should match the snapshot 1`] = `
<section
  className={undefined}
>
  <amp-img
    alt="paywall-image"
    height={900}
    layout="responsive"
    sizes={undefined}
    src="https://i.piano.io/managedservices/theeconomist/regwall-product.png"
    srcset={undefined}
    width={1600}
  />
</section>
`;
